docs/opentracing.md

```diff
@@ -51,6 +51,11 @@ docker run -d --name jaeger \
   jaegertracing/all-in-one:1
 ```
 
+By default, Synapse will publish traces to Jaeger on localhost.
+If Jaeger is hosted elsewhere, point Synapse to the correct host by setting
+`opentracing.jaeger_config.local_agent.reporting_host` in the synapse configuration
```

Comment:
Let's make this a hyperlink:

```suggestion
`opentracing.jaeger_config.local_agent.reporting_host` [in the Synapse configuration](usage/configuration/config_documentation.md#opentracing-1)
```

(I am guessing the right URL here, can you please double-check this works?)

Reply:
Also, are you able to double-check that `opentracing.jaeger_config.local_agent.reporting_host` actually works? I haven't tried it myself; I only read the jaegar client source code and guessed!

Reply:
Super, thanks for confirming.

Reply:
Also confirmed the link locally and it works like that.